doc/_templates/layout.html

```diff
@@ -0,0 +1,19 @@
+{% extends "!layout.html" %}
+
+{%- block extrahead %}
+
+   {% if theme_googletag  %}
```

Comment:
Isn't this always false? What's the point of adding this if it's never included?